crates/bevy_winit/src/lib.rs

```diff
@@ -80,7 +53,7 @@ pub static ANDROID_APP: std::sync::OnceLock<android_activity::AndroidApp> =
 /// replace the existing [`App`] runner with one that constructs an [event loop](EventLoop) to
 /// receive window and input events from the OS.
 #[derive(Default)]
-pub struct WinitPlugin {
+pub struct WinitPlugin<T: Event> {
```

Comment:
This needs docs explaining what the type parameter does.

Reply:
I added some explanation, but I think that probably a link to the example should be fine, but I did not add this though because I couldn't find the same pattern elsewhere